packages/prime-field-group/src/ui/InputComponent.tsx

```diff
@@ -68,6 +71,31 @@ export class InputComponent extends React.PureComponent<PrimeFieldProps, any> {
     this.remove(key);
   };
 
+  public onMoveUpClick = (e: React.MouseEvent<HTMLElement>) => {
+    const index = Number(e.currentTarget.dataset.index);
+    const items = this.state.items.slice(0);
+
+    if (index > 0) {
```

Comment:
maybe we can do `if (index <= 0) return` instead to prevent redundant state update,
same for the `onMoveDownClick`